packages/web3-common/src/eth_execution_api.ts

```diff
@@ -20,9 +23,8 @@ export interface AccessListEntry {
 export type AccessList = AccessListEntry[];
 export type TransactionHash = HexString;
 export type Uncles = HexString32Bytes[];
-export type BlockTag = 'earliest' | 'latest' | 'pending';
-export type BlockNumberOrTag = Uint | BlockTag;
 
+// TODO Should probably support EIP-2930 and EIP-1559
```

Comment:
@nazarhussain Do you disagree? Probably need to create an issue in ETH spec repo